lib/teamlab/modules/people/people_remove.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Teamlab
+  # Module for people removing process
+  module PeopleRemove
+    # Start a remove process
+    # @param userId [String] guid of user to remove
+    # @param [Hash] remove progress data
+    def start_remove(user_id)
```

Comment:
I think this method can remove some useful info, or nothing and in that case `@return [void]`